compose/cli/main.py

```diff
@@ -774,14 +774,14 @@ def cp(self, options):
 
         src_service, src_path = splitCpArg(source)
         dest_service, dest_path = splitCpArg(destination)
-        if src_service is not None and dest_service is not None:
+        if src_service and dest_service:
             raise UserError('copying between containers is not supported')
         service = src_service or dest_service
-        if service is None:
+        if not service:
```

Comment:
I don't think this is the exact same behavior and could lead to unwanted errors

Reply:
According to https://stackoverflow.com/questions/3901144/difference-between-if-x-and-if-x-is-not-none I think we are safe here